dlt/extract/extractors.py

```diff
@@ -318,6 +341,16 @@ def _compute_table(
             arrow_table["columns"] = pyarrow.py_arrow_to_table_schema_columns(item.schema)
             # normalize arrow table before merging
             arrow_table = self.schema.normalize_table_identifiers(arrow_table)
+            # Add load_id column
+            if (
+                self._normalize_config.add_dlt_load_id
+                and "_dlt_load_id" not in arrow_table["columns"]
```

Comment:
1. normalize _dlt_load_id before checking if in table columns
2. add column to table using schema.update_table so we are sure identifiers are properly normalized